src/components/structures/SpaceRoomView.tsx

```diff
@@ -572,10 +581,17 @@ const SpaceSetupPrivateInvite = ({ space, onFinished }) => {
             ref={fieldRefs[i]}
             onValidate={validateEmailRules}
             autoFocus={i === 0}
+            onKeyDown={ev => {
+                if (ev.key === Key.ENTER) {
+                    ev.preventDefault();
+                    onClick();
+                }
+            }}
```

Comment:
The enter key should by default work. I believe that if we were to wrap these fields on a `<form>` and set an `onSubmit` event listener on it, we would get that same "enter key" behaviour and other nice accessibility features that come from free thanks to the browser